tests/Feature/API/EndpointStreamsTest.php

```diff
@@ -0,0 +1,83 @@
+<?php
+
+namespace Tests\Feature\API;
+
+use App\Models\Stream;
+use Illuminate\Foundation\Testing\RefreshDatabase;
+use Illuminate\Support\Carbon;
+use Illuminate\Testing\Fluent\AssertableJson;
+use Tests\TestCase;
+
+class EndpointStreamsTest extends TestCase
+{
+    use RefreshDatabase;
+
+    /**
+     * @test
+     */
+    public function it_shows_all_streams(): void
+    {
+        $stream = Stream::factory()->create([
+            'title' => 'Stream #1',
+            'scheduled_start_time' => Carbon::now()->addDays(value: 2),
+            'youtube_id' => '1234',
+            'channel_title' => 'My Channel',
+            'language_code' => 'en',
```

Comment:
Please only set the values here which you are going to test against. So you can

1.) Test all provided values you list here 

or

2.) Remove the ones do you don't need for this test. (they are still being set by the default values of the factory)